cmd/statusd/main.go

```diff
@@ -27,6 +30,9 @@ var (
 	httpEnabled    = flag.Bool("http", false, "HTTP RPC endpoint enabled (default: false)")
 	httpPort       = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
 	ipcEnabled     = flag.Bool("ipc", false, "IPC RPC endpoint enabled")
+	cliEnabled     = flag.Bool("cli", false, "Enable debugging CLI server")
+	cliPort        = flag.String("cliport", debug.CLIPort, "CLI server's listening port")
+	pprofPort      = flag.String("pprof", "", "Enable profiling on port <port>")
 	logLevel       = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
```

Comment:
Here we have two similar features — `cli` and `pprof`, and different ways to configure them. For `cli` we use 2 flags (enable and port), and for `pprof` — only one (port), deducing `enable` property from `port`.
I'd suggest making it consistent — probably better to use only one flag (`cliPort`) for `cli` as well.

Reply:
Made it consistent to `httpXyz` above. Changed it before from one flag to two after review. We get into the dilemma, that reviewers may have different opinions which gets lost in GitHab reviewing.